backend/src/game.ts

```diff
@@ -100,7 +100,7 @@ function endQuestion(gameId: GameId) {
 	};
 	const hostId = game.hostId;
 	const hostSocket = userSockets.get(hostId);
-	if (hostSocket !== undefined && hostSocket.readyState === WebSocket.OPEN) {
+	if (hostSocket !== undefined && hostSocket.readyState === hostSocket.OPEN) {
```

Comment:
Is this really correct/necessary? It seems like both WebSocket.OPEN and (some WebSocket instance).OPEN exist and are identical. IMO the one on the WebSocket class itself is clearer.